filament/include/filament/Renderer.h

```diff
@@ -329,69 +259,44 @@ class UTILS_PUBLIC Renderer : public FilamentAPI {
      * readPixels() is intended for debugging and testing. It will impact performance significantly.
      *
      */
-    void readPixels(uint32_t xoffset, uint32_t yoffset, uint32_t width, uint32_t height,
+    void readPixels(RenderTarget* renderTarget,
+            uint32_t xoffset, uint32_t yoffset, uint32_t width, uint32_t height,
             backend::PixelBufferDescriptor&& buffer);
 
-
     /**
-     * Reads back the content of the provided RenderTarget.
+     * Render a standalone View into its associated RenderTarget
```

Comment:
I think `renderStandaloneView`'s signature should come after `render`, since `render` is still the preferred method (if I'm understanding correctly). Also, I think we should better document what the differences between the two are. Why would someone call `renderStandaloneView` versus `render`?